### PR TITLE
Added new files of arrays

### PR DESCRIPTION
**Pull request by allatelunts**

### CODE REVIEW

File: EnterIntegerN.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        if (arraySize > 0) {
+            int[] array = new int[arraySize];
+            if (arraySize == 15) {
+                array[0] = 15;
+            }
+            System.out.println("Array size: " + array.length);
+        } else {
+            System.out.println("Invalid input. Exiting.");
+            System.exit(1);
+        }
```
Comment (David91) [inline EnterIntegerN.java]: This is part of the requirement:
"Խնդիրը լուծելուց չեք օգտագործում պայմանի օպերատոր։։"

File: FiveElementsArray.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        if (originalArray[1] >= 0) nonNegativeArray[count++] = originalArray[1];
+        if (originalArray[2] >= 0) nonNegativeArray[count++] = originalArray[2];
+        if (originalArray[3] >= 0) nonNegativeArray[count++] = originalArray[3];
+        if (originalArray[4] >= 0) nonNegativeArray[count++] = originalArray[4];
+
+        if (count > 0) reversedArray[0] = nonNegativeArray[count - 1];
+        if (count > 1) reversedArray[1] = nonNegativeArray[count - 2];
+        if (count > 2) reversedArray[2] = nonNegativeArray[count - 3];
+        if (count > 3) reversedArray[3] = nonNegativeArray[count - 4];
+        if (count > 4) reversedArray[4] = nonNegativeArray[count - 5];
```
Comment (David91) [inline FiveElementsArray.java]: Couldn't this be solved by using only 1 array instead of 2?

File: FixedSizeIntegerArray.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            larger = b;
+        } else {
+            smaller = b;
+            larger = a;
+        }
+
+        int middleIndex = n / 2;
+        if (n > 1) {
+            array[middleIndex - 1] = smaller;
+            if (middleIndex < n) {
```
Comment (David91) [inline FixedSizeIntegerArray.java]: Why is this condition? Is there any probability, that `middleIndex` can more than equal to `n` ?

**Review by David91 — CHANGES_REQUESTED** [review]

File: EnterIntegerN.java (hunk 1 above)
Comment (allatelunts) [reply EnterIntegerN.java]: The comment is fixed, instead of the condition operator I used 'switch'

File: FiveElementsArray.java (hunk 2 above)
Comment (allatelunts) [reply FiveElementsArray.java]: The comment is fixed, I used one array

File: FixedSizeIntegerArray.java (hunk 3 above)
Comment (allatelunts) [reply FixedSizeIntegerArray.java]: The comment is fixed: the condition  is corrected

**Pull request merged**